src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -43,8 +43,9 @@ trait Backend {
    * Return a possibly altered copy of inputs reflecting any localization of input file paths that might have
    * been performed for this `Backend` implementation.
    */
-  def adjustInputPaths(call: Call, inputs: CallInputs): CallInputs
+  def adjustInputPaths(callKey: CallKey, inputs: CallInputs, workflowDescriptor: WorkflowDescriptor): CallInputs
 
+  // FIXME: This is never called...
```

Comment:
Cool, works for me. I made a note in the main description just so ppl don't miss it